### DISCUSSION TITLE
Re-enable GitHub Issues for the meshrdf repo

### DISCUSSION THREAD

**Discussion by dhimmel** [HHS/meshrdf#206] [General]
Am a big fan and user of MeSH RDF. I love the great docs and that the project is on GitHub. We make use of the project heavily in  https://github.com/related-sciences/nxontology-data.

I just noticed that it appears GitHub Issues have been disabled for this repo. And possibly discussions enabled in their place? Unfortunately, this took all existing issues offline. For example, our code links to the following issues which are now offline:

```
https://github.com/HHS/meshrdf/issues/150
https://github.com/HHS/meshrdf/issues/155
https://github.com/HHS/meshrdf/issues/193
https://github.com/HHS/meshrdf/issues/194
```

Would it be possible to re-enable issues, as there are likely 100s of hours of work captured by these issues and they form an important part of the scholarly record? If the goal is to not allow new issues, perhaps there's a way we could export a public archive of all existing issues?

**Comment by fallgrennj** [comment]
Thanks very much for letting us know about this problem.  

I researched your question and learned that the prior leads of MeSH RDF disabled GitHub issues not realizing that they would lose all pre-existing issues, i.e., they thought the existing issues would remain available/visible but that no new issues would be allowed to be entered.  At that time, users were asked to send help requests to an internal service at https://support.nlm.nih.gov/, which is still in use.

That said, the programmer who set up this GitHub account is no longer working at NLM, and while I am currently a co-lead for MeSH RDF, I am not an owner of this account.  If I can discover who owns the account, I will see if issues can be re-enabled so that you can at least see and perhaps copy the ones you need - although we do not want to encourage anyone to put any new MeSH RDF issues in GitHub.  Creating a public archive for old GitHub issues may be a reasonable suggestion, but we will need to investigate all options.  

In addition, you should know that MeSH RDF is basically static at this point in time except for annual updates, which are available to download from https://id.nlm.nih.gov/mesh/ 

I'm sorry for the inconvenience this has inadvertently caused.
-Nancy Fallgren
Metadata Specialist Librarian, NLM

**Reply by fallgrennj** [reply]
@dhimmel  We've re-enabled Issues so that you may see them and perhaps download those you need to keep. 

We really don't want anyone adding new issues on GitHub, so will be considering a means for keeping the existing issues publicly visible, but inactive - perhaps in a public archive as you suggest. 

Thanks again for bringing this to our attention.

**Reply by dhimmel** [reply]
Thanks @fallgrennj for the help and sounds sounds like you were able to get admin access to @HHS figured out.

> In addition, you should know that MeSH RDF is basically static at this point in time except for annual updates

Aside: if you ever would like a letter of support attesting to the usefulness of MeSH RDF, happy to provide one.

Do you envision making any future commits to this repo? If not, you could [archive](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories) it, which basically puts everything in a read only state. No new issues would be allowed nor would commenting on old issues.

It is too bad [disabling issues](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/disabling-issues) removes existing issues. Would be nice if there was a way to lock all existing issues and not allow new issues. One workaround could be locking existing issues, especially where conversion appears complete and then using an [issue template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) that says something like "Please don't open an issue, use https://support.nlm.nih.gov/ instead".

> users were asked to send help requests to an internal service at https://support.nlm.nih.gov/, which is still in use.

I'm a big fan of public issue trackers because they deduplicate effort and contribute to a more transparent and open & collaborative scientific discourse. With the private issue tracker, the same question is bound to get asked by many users. Anyways this policy is probably outside of your control, so understand that our conversation here is unlikely to change it.

Finally noting the README could be updated about the current support recommendation:

https://github.com/HHS/meshrdf/blob/45858568755c357c00b3f1900463ef209579f7e2/README.md#L17

Thanks for the help! I'll follow up with anything I find about third-party archiving of GitHub issues.
